server/app/views/JsBundle.java

```diff
@@ -0,0 +1,16 @@
+package views;
+
+public enum JsBundle {
```

Comment:
suggestion (blocking): please add java doc for this that mentions the difference between different bundles

Reply:
Done.